Yosemite/YosemiteTests/Stores/AnnouncementsStoreTests.swift

```diff
@@ -179,30 +177,17 @@ private extension AnnouncementsStoreTests {
         FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first?.appendingPathComponent("feature-announcements.plist")
     }
 
-    func makeWordPressAnnouncement() throws -> WordPressKit.Announcement {
-        let jsonData = try JSONSerialization.data(withJSONObject: [
-            "appVersionName": "1",
-            "minimumAppVersion": "",
-            "maximumAppVersion": "",
-            "appVersionTargets": [],
-            "detailsUrl": "http://wordpress.org",
-            "features": [[
-                "title": "foo",
-                "subtitle": "bar",
-                "icons": [[
-                    "iconUrl": "https://s0.wordpress.com/i/store/mobile/plans-premium.png",
-                    "iconBase64": "",
-                    "iconType": ""
-                ]],
-                "iconBase64": "",
-                "iconUrl": "https://s0.wordpress.com/i/store/mobile/plans-premium.png"
-            ]],
-            "announcementVersion": "2",
-            "isLocalized": true,
-            "responseLocale": "en_US"
-        ])
-
-        return try JSONDecoder().decode(Announcement.self, from: jsonData)
+    func makeWordPressAnnouncement() -> Yosemite.Announcement {
```

Comment:
nit: We could remove the reference to WordPress in the method name, since this isn't coming from WPKit anymore.

Reply:
Good point - done in 8de0a9d85b223b13f12d813bc0ba68bbf961bb94.